docs/index.rst

```diff
@@ -42,10 +41,10 @@ Now you need to set up Raven.js to use your `Sentry DSN
 At this point, Raven is ready to capture any uncaught exception.
 
 Once you have Raven up and running, it is highly recommended to check out :doc:`config`
```

Comment:
maybe we should say "for information about configuration options and advanced usage"